packages/eslint-plugin/src/index.js

```diff
@@ -9,13 +9,16 @@
 
 import validStyles from './stylex-valid-styles';
 import sortKeys from './stylex-sort-keys';
+import marginPadding from './stylex-margin-padding-shorthand';
 
 const rules: {
   'valid-styles': typeof validStyles,
   'sort-keys': typeof sortKeys,
+  'margin-padding': typeof marginPadding,
```

Comment:
Can we call this `valid-shorthands` so we can add the other shorthands later on

Reply:
Yes!